--- cookbooks/openstack-compute/recipes/nova-setup.rb-orig	2015-03-26 23:52:31.805062744 -0400
+++ cookbooks/openstack-compute/recipes/nova-setup.rb	2015-03-26 01:26:26.784933948 -0400
@@ -83,7 +83,7 @@
   if floating && (floating['ipv4_cidr'] || floating['ipv4_range'])
     cmd = ''
     if floating['ipv4_cidr']
-      cmd = "#{node['openstack']['compute']['floating_cmd']} nova --cidr=#{floating['ipv4_cidr']}"
+      cmd = "#{node['openstack']['compute']['floating_cmd']} nova --pool default --cidr=#{floating['ipv4_cidr']}"
     elsif floating['ipv4_range']
       cmd = "#{node['openstack']['compute']['floating_cmd']} nova --ip-range=#{floating['ipv4_range']}"
     end
